WcaOnRails/db/structure.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.16  Distrib 10.1.9-MariaDB, for Linux (x86_64)
+-- MySQL dump 10.13  Distrib 5.5.46, for debian-linux-gnu (x86_64)
```

Comment:
It looks like nothing actually changed here, please remove this file from the commit.